stable/phpmyadmin/templates/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "phpmyadmin.fullname" . }}
+  labels:
+    app: {{ template "phpmyadmin.name" . }}
+    chart: {{ template "phpmyadmin.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "phpmyadmin.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "phpmyadmin.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ template "phpmyadmin.image" . }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: DATABASE_PORT_NUMBER
+              value: "{{ .Values.phpmyadmin.dbPort }}"
+            {{- if .Values.phpmyadmin.dbSuffix }}
+            - name: DATABASE_HOST
+              value: "{{ template "db.fullname" . }}"
+            {{- else }} 
+            - name: DATABASE_HOST
+              value: "{{ .Values.phpmyadmin.dbHost }}"
+            {{- end }}
+            {{- if and (not .Values.phpmyadmin.dbSuffix) (not .Values.phpmyadmin.dbHost)}}
+            - name: PHPMYADMIN_ALLOW_NO_PASSWORD
+              value: "true"
+            - name: PHPMYADMIN_ALLOW_ARBITRARY_SERVER
+              value: "true"
+            {{- else }}
+            - name: PHPMYADMIN_ALLOW_NO_PASSWORD
+              value: "false"
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+            - name: https
+              containerPort: 443
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
```

Comment:
would be nice to be able to enable/disable the  probes?

Reply:
it wont hurt I guess